meson.build

```diff
@@ -169,4 +169,16 @@ if get_option('docs')
   subdir('doc')
 endif
 
-meson.add_install_script('build-aux/post-install.sh')
+gnome = import('gnome')
+if find_program('update-desktop-database', required: false).found()
```

Comment:
This would probably be better without the extra conditional:

```
update_desktop_database = find_program('update-desktop-database', required: false)
gnome.post_install(
  glib_compile_schemas: true,
  gtk_update_icon_cache: true,
  update_desktop_database: update_desktop_database.found(),
)
```

Reply:
Thanks, that's a much better way to do it!